### ISSUE TITLE
Default/Example Row for DocumentationTool

### ISSUE THREAD

**Issue by carla-terboven** [nomad-hzb/nomad-baseclasses#48]
Especially new users struggle with putting valid data in the CE-NOME_Documentation_Tool. Strings, numbers or units in the wrong places make the generation of IDs impossible. It would be nice to have a default row in the beginning where users can only modify specific values while seeing the correct format.

**Comment by RoteKekse** [comment]
yes this can easily be done good idea and then this row is always skipped, i cant make the row in editable though, 

we also can add some description at the top, defining default values with markdown
